substrate/frame/fast-unstake/src/tests.rs

```diff
@@ -788,10 +788,12 @@ mod on_idle {
 			assert_ok!(FastUnstake::register_fast_unstake(RuntimeOrigin::signed(VALIDATOR_PREFIX)));
 
 			// but they indeed are exposed!
-			assert!(pallet_staking::ErasStakers::<T>::contains_key(
+			assert!(pallet_staking::EraInfo::<T>::get_paged_exposure(
 				BondingDuration::get() - 1,
-				VALIDATOR_PREFIX
-			));
+				&VALIDATOR_PREFIX,
+				0
+			)
+			.is_some());
```

Comment:
It seems to be that probably we could have prevented this issue if instead of an `assert!` there was an `assert_eq!` that would check if the exposure is correct. Maybe a good time to add that?

Reply:
What do you mean? Checking the exposure?

Reply:
>  Checking the exposure?
yes, the return value of `fn get_paged_exposure`